test/unit/nft-api.test.ts

```diff
@@ -1260,6 +1261,45 @@ describe('NFT module', () => {
     });
   });
 
+  describe('summarizeNftAttributes()', () => {
+    const contractAddress = '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d';
+    const templateResponse: NftAttributesSummary = {
+      contractAddress,
+      totalSupply: 1000,
+      summary: {
+        Background: {
+          Aquamarine: 3,
+          'New Punk Blue': 5
+        },
+        Eyes: {
+          Cyborg: 1,
+          Zombie: 7
+        }
+      }
+    };
+
+    it('calls with the correct parameters', async () => {
+      mock.onGet().reply(200, templateResponse);
+
+      await alchemy.nft.summarizeNftAttributes(contractAddress);
+
+      expect(mock.history.get.length).toEqual(1);
+      expect(mock.history.get[0].params).toHaveProperty(
+        'contractAddress',
+        contractAddress
+      );
```

Comment:
can you add a few checks to verify that the response contains all the properties?

Reply:
Sure 👍🏼